src/vector/feature.rs

```diff
@@ -87,4 +150,12 @@ impl FieldValue {
             _ => panic!("not a RealValue")
         }
     }
+
+    /// Interpret the value as `i32`. Panics if the value is something else.
+    pub fn as_int(self) -> i32 {
```

Comment:
I made the other changes but not this last one (*returning an `Option` instead of panicking*), because I didn't know how to handle this when using a `FieldValue` to set a field :
```rust
pub fn set_field(&self, field_name: &str, type_value: OGRFieldType, value: FieldValue) -> Result<()> {
    match type_value {
        OGRFieldType::OFTReal => self.set_field_double(field_name, value.to_real()),
```
like here, when using  `value.to_real()`. I guess there is a way to handle this nicely without using `unwrap`, i will look into the `error_chain` crate documentation.